platform/core/package.json

```diff
@@ -31,7 +31,7 @@
   },
   "peerDependencies": {
     "cornerstone-math": "0.1.9",
-    "cornerstone-wado-image-loader": "^4.2.1",
+    "cornerstone-wado-image-loader": ">=4.8.0",
```

Comment:
Ditto

Reply:
I just changed the version. Are you confusing with "@cornerstonejs/streaming-image-volume-loader"?